pebble/README.md

```diff
@@ -0,0 +1,83 @@
+# Pebble
+
+A fast key-value DB using [cockroachdb/pebble](https://github.com/cockroachdb/pebble)
+
+### Table of Contents
+
+- [Signatures](#signatures)
+- [Installation](#installation)
+- [Examples](#examples)
+- [Config](#config)
+- [Default Config](#default-config)
+
+### Signatures
+
+```go
+func New(config ...Config) Storage
+func (s *Storage) Get(key string) ([]byte, error)
+func (s *Storage) Set(key string, val []byte, exp time.Duration) error
+func (s *Storage) Delete(key string) error
+func (s *Storage) Reset() error
+func (s *Storage) Close() error
+func (s *Storage) Conn() *badger.DB
+```
+
+### Installation
+
+Pebble is tested on the 2 last [Go versions](https://golang.org/dl/) with support for modules. So make sure to initialize one first if you didn't do that yet:
+
+```bash
```

Comment:
Add a small comment here saying this is only if you dont have an existing module